app/AppKernel.php

```diff
@@ -200,8 +200,8 @@ public function registerBundles(): iterable
         }
 
         // Check for local bundle inclusion
-        if (file_exists($this->getProjectDir().'/app/config/bundles_local.php')) {
-            include $this->getProjectDir().'/app/config/bundles_local.php';
+        if (file_exists($this->getProjectDir().'/local_config/bundles_local.php')) {
```

Comment:
If I get it right the file `/app/config/bundles_local.php`  may still exist and may be in different tutorials, the users are free to create it, but it just won't work as the user expects. 
I suggest a deprecation exception, it should not run if these files exist. it may happen the upgrade.php won't be run at all.
something like: if count(app/config/local_*) > exception('you need to migrate your configuration');

Reply:
@galvani the idea is good, but that line would also be triggered by the doctrine migrations moving the files. 

I'd go for logging a warning when there are still `app/config/local_*` files present